README.md

```diff
@@ -1 +1,4 @@
-# codebreker_rack
\ No newline at end of file
+Codebreaker Rack
+
+Heroku link:
+https://codebreakerrackmanfly.herokuapp.com/
```

Comment:
In fact, I see a lot of differences in the documentation (button names, etc.) and errors in behavior. Check the app for this

Reply:
https://i.imgur.com/qBFO7Ki.png

Reply:
I see several js errors in the console. Please check the file routes